### PR TITLE
50 travis

### PR DESCRIPTION
**Pull request by madziaf**
Ataru can run on Travis.
`Gogo` method changed to `check`, which can take arguments (markdown files) or, if no argument has been passed, it loads all the .md files of the project.
Changed gem version to 0.1.0 in order to push it to RubyGems.

### CODE REVIEW

File: lib/ataru/cli/application.rb [hunk 1]
```diff
@@ -27,8 +27,11 @@ def setup
         puts "hi dude, change the created ataru_setup.rb file adding your dependencies needed for your code snippets to run"
       end
 
-      desc "gogo FILENAMES", "Run and validate code samples in given markdown files"
-      def gogo(*filenames)
+      desc "check", "Check code snippets in all .md files of the project or, if given, specific .md files"
+      def check(*filenames)
+        if filenames.length == 0
+          filenames = Dir.glob('**/*md')
```
Comment (moonglum) [inline lib/ataru/cli/application.rb:28]: Nice! That's a cool trick :+1: I will put that into my toolbelt!

**Comment by moonglum** [comment]
README looks very good (apart from the small remarks in HipChat) :+1:

**Comment by madziaf** [comment]
solves #50 
solves #67

**Comment by moonglum** [comment]
Awesome :dancers:

**Comment by schultyy** [comment]
Wunderbar :)

File: ataru.gemspec [hunk 2]
```diff
@@ -5,7 +5,7 @@ require 'ataru/version'
 
 Gem::Specification.new do |spec|
   spec.name          = "ataru"
-  spec.version       = "0.0.1"
+  spec.version       = "0.1.0"
```
Comment (kazjote) [inline ataru.gemspec:8]: :cake:

**Comment by madziaf** [comment]
:dancers:

**Comment by madziaf** [comment]
OMG! what are we going to do now? ;)

**Pull request merged**